### PR TITLE
feature/GH-13212: move order style into order lib

### LINKED ISSUE #13212: move order style into order lib

### PR DESCRIPTION
**Pull request by WeizhengSap**
close #13212

### CODE REVIEW

**Review by plabadie — CHANGES_REQUESTED** [review]
I think we have to revert all the changes to the existing styles because this is a breaking change.

What I do recommend we do, is add a deprecated comment at each place we need to delete something in 5.0.  I don't think we'll remember what to remove when get there + it's a good thing to announce our intentions in advance like we do with the rest.

File: projects/storefrontstyles/scss/components/content/tab-paragraph-container/_tab-paragraph-container.scss [hunk 1]
```diff
@@ -195,4 +195,8 @@ $useTabsOnPDP: false !default;
   } @else {
     @extend %pdpFlat !optional;
   }
+
+  .active {
+    background-color: var(--cx-color-inverse);
+  }
```
Comment (plabadie) [inline projects/storefrontstyles/scss/components/content/tab-paragraph-container/_tab-paragraph-container.scss:201]: Is that a fix in the olde styles?

File: projects/storefrontstyles/scss/layout/page-templates/_order-detail-page.scss [hunk 2]
```diff
@@ -89,16 +93,11 @@
...
   cx-order cx-order-details-totals {
     margin-bottom: 32px;
   }
 
-  cx-tab-paragraph-container {
-    .active {
-      background-color: var(--cx-color-inverse);
-    }
-  }
-
```
Comment (plabadie) [inline projects/storefrontstyles/scss/layout/page-templates/_order-detail-page.scss:101]: Why do we remove this part?  It's still a breaking change I think.

Comment (WeizhengSap) [reply projects/storefrontstyles/scss/layout/page-templates/_order-detail-page.scss:101]: It is moved to tab-paragraph-container.scss

File: projects/storefrontstyles/scss/components/content/tab-paragraph-container/_tab-paragraph-container.scss (hunk 1 above)
Comment (WeizhengSap) [reply projects/storefrontstyles/scss/components/content/tab-paragraph-container/_tab-paragraph-container.scss:201]: it is moved from `_order-detail-page.scss ` to here.

**Review by plabadie — APPROVED** [review]

**Pull request merged**